app/v2/notifications/post_notifications.py

```diff
@@ -161,6 +161,11 @@ def post_bulk():
         request_json = request.get_json()
     except werkzeug.exceptions.BadRequest as e:
         raise BadRequestError(message=f"Error decoding arguments: {e.description}", status_code=400)
+    except werkzeug.exceptions.UnsupportedMediaType as e:
+        raise BadRequestError(
+            message="UnsupportedMediaType error: {}".format(e.description),
+            status_code=415,
+        )
```

Comment:
I had thought we should add some details here, but TBH it's probably better to just pass the error on appropriately as you do here until we hear otherwise from our users.

Reply:
actually in my local test I get the error message
```
"UnsupportedMediaType error: Did not attempt to load JSON data because the request Content-Type was not 'application/json'."
```
which does totally explain the problem! 👍